From 385bf1c2ec57942e17ee529e57eef0dcd99904e6 Mon Sep 17 00:00:00 2001
From: Roy Li <rongqing.li@windriver.com>
Date: Tue, 1 Sep 2015 17:00:33 +0800
Subject: [PATCH] [PATCH] conditionally check libvirt

Upstream-Statue: Pending

check if libvirt is available only when a user wants to use libvirt

Signed-off-by: Roy Li <rongqing.li@windriver.com>

---
 configure.ac | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 101d6f9f..a7eca97d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5758,11 +5758,14 @@ else
   with_libxml2="no (pkg-config doesn't know libxml-2.0)"
 fi
 
-$PKG_CONFIG --exists libvirt 2>/dev/null
-if test $? = 0; then
-  with_libvirt="yes"
-else
-  with_libvirt="no (pkg-config doesn't know libvirt)"
+if test "x$enable_libvirt" = "xyes"; then
+	$PKG_CONFIG --exists libvirt 2>/dev/null
+	if test "$?" = "0"
+	then
+		with_libvirt="yes"
+	else
+		with_libvirt="no (pkg-config doesn't know libvirt)"
+	fi
 fi
 
 if test "x$with_libxml2" = "xyes"; then
